lib/config/hardware-config.ts

```diff
@@ -263,18 +263,45 @@ export function getWPILibMotorType(motorName: string): string {
     case "NEO":
       return "DCMotor.getNEO(1)"
     case "NEO550":
-      return "DCMotor.getNEO550(1)"
-    case "Minion":
-      return "DCMotor.getMinion(1)"
-    case "Krakenx44":
-      return "DCMotor.getKrakenX44(1)"
+      return "DCMotor.getNeo550(1)"
     case "Krakenx60":
       return "DCMotor.getKrakenX60(1)"
+    case "Minion":
+    case "Krakenx44":
+      return "custom" // Does not exist in WPILib, use output of getCustomDCMotorMethod
     default:
       return "DCMotor.getNEO(1)"
   }
 }
 
+export function getCustomDCMotorMethod(motorName: string): string {
```

Comment:
This seems like it doesn't need to exist when we could just create the motors in the other function as a `new DCMotor(...)`? 

Seems like this is more code than necessary?

Reply:
@thenetworkgrinch Updated! I reworked it be in-line, and made a small helper method to reduce the template string duplication we'd have to do - I like how it turned out.